sdcm/cluster_k8s/__init__.py

```diff
@@ -2641,8 +2641,12 @@ def replace_scylla_cluster_value(self, path: str, value: Any,
                                      dc_idx: int = 0) -> Optional[ANY_KUBERNETES_RESOURCE]:
         self.k8s_clusters[dc_idx].log.debug(
             "Replace `%s' with `%s' in %s's spec", path, value, self.scylla_cluster_name)
+        request_body = [{"op": "replace", "path": path, "value": value}]
+        # Scylla >= 2025.x images are published to scylladb/scylla instead of scylladb/scylla-enterprise
+        if path.__contains__("version") and ComparableScyllaVersion(value) >= "2025.1.0":
+            request_body.append({"op": "replace", "path": "/spec/repository", "value": "scylladb/scylla"})
         return self._k8s_scylla_cluster_api(dc_idx=dc_idx).patch(
-            body=[{"op": "replace", "path": path, "value": value}],
+            body=request_body,
             name=self.scylla_cluster_name,
```

Comment:
Will it be needed if recent versions (`2025.1+`) are used in all of the scylla-operator pipelines [1] ?
[1] https://github.com/scylladb/scylla-cluster-tests/pull/12901